src/bcf-toolkit/Builder/Bcf30/DocumentReferenceBuilder.cs

```diff
@@ -18,13 +20,24 @@ public partial class DocumentReferenceBuilder :
     return this;
   }
 
+  public DocumentReferenceBuilder SetDocumentGuid(string? guid) {
+    if (guid != null)
```

Comment:
it depends on what you are used to, with a 1-line condition it is more understandable for me if there is no bracket:)
this can be another solution without if:
    _documentReference.DocumentGuid = guid ?? _documentReference.DocumentGuid;